CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## TBD
+
+This release includes new versions of both bugsnag-android and
+bugsnag-android-ndk, which will be released together going forward.
```

Comment:
It'd be good to clarify whether this will be as one individual artefact, or bundled together.